website/content/v1.9/talos-guides/configuration/image-cache.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Image Cache"
+description: "How to enable and configure Talos image cache feature."
+---
+
+Talos Image Cache feature allows to provide container images to the nodes without the need to pull them from the Internet.
+This feature is useful in environments with limited or no Internet access.
+
+Image Cache is local to the machine, and automatically managed by Talos if enabled.
+
+## Preparing Image Cache
+
+First, build a list of image references that need to be cached.
+The `talosctl images default` might be used as a starting point, but it should be customized to include additional images (e.g. custom CNI, workload images, etc.)
+
+```bash
+talosctl images default > images.txt
+cat extra-images.txt >> images.txt
+```
+
+Next, prepare an OCI tarball which contains all cached images:
+
+```bash
+cat images.txt | talosctl images cache-create --image-cache-path image-cache.tar --images=-
+```
+
+> Note: The `cache-create` supports a `--layer-cache` flag to additionally cache the pulled images layers on the filesystem.
+> This is useful to speed up repeated calls for `cache-create` with the same images.
+
+The image cache tarball might be used as a file (`image-cache.tar`) or pushed itself to a container registry of your choice.
+
+## Building Boot Assets
+
+The image cache is provided to Talos via the boot assets.
+There are two supported boot asset types for the Image Cache: ISO and disk image.
+
+### ISO
+
+In case of ISO, the image cache is bundled with a Talos ISO image, it will be available for the initial install and (if configured) copied to the
+disk during the installation process.
+
+The ISO image can built with the [imager]({{< relref "../install/boot-assets#imager" >}}) by passing an additional `--image-cache` flag:
+
+```bash
+docker run --rm -t -v $PWD/_out:/secureboot:ro -v $PWD/_out:/out -v /dev:/dev --privileged ghcr.io/siderolabs/imager:{{< release >}} iso --image-cache ./image-cache.tar
```

Comment:
probably add a note that `_out` should contain the folder

Reply:
added `mkdir`